x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/snapshot_policies/snapshot_policies.tsx

```diff
@@ -0,0 +1,157 @@
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import {
+  EuiButtonIcon,
+  EuiCallOut,
+  EuiComboBox,
+  EuiComboBoxOptionOption,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { useLoadSnapshotPolicies } from '../../../../services/api';
+
+interface Props {
+  value: string;
+  onChange: (value: string) => void;
+}
+export const SnapshotPolicies: React.FunctionComponent<Props> = ({ value, onChange }) => {
+  const { error, isLoading, data, sendRequest } = useLoadSnapshotPolicies();
+
+  const policies = data.map((name: string) => ({
+    label: name,
+    value: name,
+  }));
+
+  const onComboChange = (options: EuiComboBoxOptionOption[]) => {
+    if (options.length > 0) {
+      onChange(options[0].label);
+    } else {
+      onChange('');
+    }
+  };
+
+  const onCreateOption = (newValue: string) => {
+    onChange(newValue);
+  };
+
+  let calloutContent;
+  if (error) {
+    calloutContent = (
+      <Fragment>
+        <EuiSpacer size="m" />
+        <EuiCallOut
+          data-test-subj="policiesErrorCallout"
+          size="s"
+          iconType="help"
+          color="warning"
+          title={
+            <Fragment>
+              <FormattedMessage
+                id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesLoadedTitle"
+                defaultMessage="Couldn't load existing policies."
+              />
+
+              <EuiButtonIcon
+                size="s"
+                color="warning"
+                onClick={sendRequest}
+                iconType="refresh"
+                aria-label={i18n.translate(
+                  'xpack.indexLifecycleMgmt.editPolicy.deletePhase.reloadPoliciesLabel',
+                  {
+                    defaultMessage: 'Reload policies',
+                  }
+                )}
+              />
+            </Fragment>
+          }
+        >
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesLoadedMessage"
+            defaultMessage="You still can type in a policy name."
+          />
+        </EuiCallOut>
+      </Fragment>
+    );
+  } else if (data.length === 0) {
+    calloutContent = (
+      <Fragment>
+        <EuiSpacer size="m" />
+        <EuiCallOut
+          data-test-subj="noPoliciesCallout"
+          size="s"
+          iconType="help"
+          color="warning"
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesCreatedTitle"
+              defaultMessage="No policies"
+            />
+          }
+        >
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesCreatedMessage"
+            defaultMessage="You haven't created any snapshot policies yet. You still can type in a policy name."
```

Comment:
```suggestion
            defaultMessage="Enter a name to use for a new snapshot policy."
```

Reply:
This case is similar to a custom policy name option, the user can type in any value, but they will have to navigate to "Snapshot and Restore" and create a snapshot policy themselves. I'm planning to add a link to the callout later. I was hoping you could help me with a clear wording for this case. 
<img width="435" alt="Screenshot 2020-07-08 at 14 52 36" src="https://user-images.githubusercontent.com/6585477/86921062-d5eba980-c12a-11ea-95db-3e96478515c0.png">

Reply:
Ah, I see. If users can enter a name but still need to navigate away from this screen to create a snapshot policy, then this description might be more sensible:

>**No snapshot policies found**
>[Create a snapshot lifecycle policy](https://www.elastic.co/guide/en/kibana/7.8/snapshot-repositories.html#kib-snapshot-policy) to automate the creation and deletion of cluster snapshots.

Reply:
👍